Slack-Apps/week1/homework/responses/1.3-setup-app-and-learn-events.md

```diff
@@ -0,0 +1,92 @@
+# Setting up a Slack App
+
+Now that you have done some research on Slack development, its time to get building! In this issue, you will learn how to set up a Slack app.
+
+## Installing a Text Editor
+
+Since your slack app will be runing locally, you will need a place to store and edit your code. You can create a folder anywhere within your file system for this project. If you do not have one already, it is highly reccomended that you download an IDE or text editor. This will give you a better interface for when you program, helping you to notice errors and run your code efficiently. Some good editors include [VSCode](https://code.visualstudio.com/download) or [Atom](https://atom.io/).
+
+## Creating a Slack App
+
+In order to create the app, you will need to [create a Slack account](https://slack.com/get-started#/create) and then make a [make a new app](https://api.slack.com/apps?new_app=1) on the developer platform. Once you have made the app, you should be able to click on it and navigate to a page similar to the one below:
+
+![Slack app home](./Gifs_Images/basic-info-page.png)
```

Comment:
not a big deal, but fix the images :)

Reply:
Hey @FifiTeklemedhin, can you fix these changes?

Reply:
Got it @Bahburs! Thanks @emsesc for the feedback.

Reply:
@emsesc the images seem fine in my repo, what should be fixed?

Reply:
Sorry about that, just double checked and they look fine in your repo. Probably because I was looking at it through the PR. My bad, your images are fine :)